wunderio/client-UK-FSA-beta
# FSA-1034: Adjust flex property for IE

To test:
- click 'Contact' from the homepage
- check the three li flex items ('report a food problem' etc) are full width on ie11
- double check across browsers just in case (tested in chrome/ff/ie)
